build/Targets/DatabaseBuild.cs

```diff
@@ -40,29 +44,48 @@ partial class Build
     readonly bool RollbackMigration = false;
 
     Target UpdateDatabase => _ => _
-        .DependsOn(DropDatabase)
         .Executes(() =>
         {
-            EntityFrameworkDatabaseUpdate(_ => _
-                .SetProcessWorkingDirectory(SourceDirectory)
-                .SetMigration(RollbackMigration ? "0" : (UpdMigrName ?? String.Empty))
-                .SetProject(@"Streetcode.DAL\Streetcode.DAL.csproj")
-                .SetStartupProject(@"Streetcode.WebApi\Streetcode.WebApi.csproj")
-                .SetContext("Streetcode.DAL.Persistence.StreetcodeDbContext")
-                .SetConfiguration(Configuration)
+            DotNetRun(s => s
+            .SetProjectFile(DbUpdateDirectory)
+            .SetConfiguration(Configuration)
             );
         });
 
-    Target DropDatabase => _ => _
+    Target CreateSQLScripts => _ => _
         .Executes(() =>
         {
-            EntityFrameworkDatabaseDrop(_ => _
-                .SetProcessWorkingDirectory(SourceDirectory)
-                .EnableForce()
-                .SetProject(@"Streetcode.DAL\Streetcode.DAL.csproj")
-                .SetStartupProject(@"Streetcode.WebApi\Streetcode.WebApi.csproj")
-                .SetContext("Streetcode.DAL.Persistence.StreetcodeDbContext")
-                .SetConfiguration(Configuration)
-            );
+            Console.WriteLine("Select name of SQL script:");
+            string queryName = Console.ReadLine();
+
+            Console.WriteLine("Select SEED or MIGRATE SQL script:");
+            string queryType = Console.ReadLine();
+
+            if (queryType == "SEED")
+            {
+                queryType = "ScriptsSeeding";
+            } 
+            else 
+            {
+                queryType = "ScriptsMigration";
+            }
+
+            var dbPath = Directory.GetCurrentDirectory() + @"\Streetcode.WebApi";
+            var outputScriptPath = Directory.GetCurrentDirectory() + @$"\Streetcode.DAL\Persistence\{queryType}\";
+
+            PowerShell(@$"dotnet ef migrations script --idempotent --output {outputScriptPath}{queryName}.sql  --project {dbPath}");
```

Comment:
Let's add here dependent step that will install ef globally, as right now if you don't have it this command will fail
The same dependent step let's add to CreateDatabase target